packages/admin/cms-admin/src/common/header/impersonation/UserHeaderAccountIcon.tsx

```diff
@@ -0,0 +1,46 @@
+import { Account } from "@comet/admin-icons";
+import { styled } from "@mui/material/styles";
+
+interface UserHeaderAccountIconProps {
+    impersonated: boolean;
+}
+
+export function UserHeaderAccountIcon({ impersonated }: UserHeaderAccountIconProps) {
+    return (
+        <AccountIconWrapper>
+            <AccountIcon active={impersonated}>
+                <Account color="inherit" />
+            </AccountIcon>
+            {impersonated && (
+                <ImpersonatedIcon active={!impersonated}>
+                    <Account color="inherit" />
+                </ImpersonatedIcon>
+            )}
+        </AccountIconWrapper>
+    );
+}
+
+const AccountIcon = styled("div", {
+    shouldForwardProp: (prop) => prop !== "active",
+})<{ active?: boolean }>`
```

Comment:
Should we use AvatarGroup for this? https://mui.com/material-ui/react-avatar/#grouped